kevinhartmann23/fitlit
# Add blur effect to account card display

#### What does  this PR do?

This PR adds a blurring effect on account information display.

#### Where should the reviewer start?

Familiarize yourself with the blur function added to scripts.js.

#### Is this a bug fix or a feature?

Feature.

#### How should this be manually tested?

In browser, open account information to ensure that blur effect is functioning properly.
